bindings/core/src/method/utils.rs

```diff
@@ -163,9 +164,29 @@ pub enum UtilsMethod {
     #[serde(rename_all = "camelCase")]
     OutputHexBytes { output: OutputDto },
     /// Verifies the semantic of a transaction.
+    /// Expected response: [`ConflictReason`](crate::Response::ConflictReason)
     VerifyTransactionSemantic {
         inputs: Vec<InputSigningDataDto>,
         transaction: TransactionPayloadDto,
         time: u32,
     },
+    /// Verifies the syntax of a transaction.
+    /// Expected response: [`Ok`](crate::Response::Ok)
+    #[serde(rename_all = "camelCase")]
+    VerifyTransactionSyntactic {
```

Comment:
Why not `VerifyTransactionSyntax`?

Reply:
Yeah probably better actually, I'll change it